amp/repository/aim/view/teamAuditList.jsp

```diff
@@ -20,6 +20,12 @@ function showUser(email){
 	}
 }
 
+function compare(id){

```

Comment:
this method is similar to the one in auditLoggerManager.jsp lets extract this into a .js file (you might need to add the form as a parameter) so we avoid having dupe code